README.md

```diff
@@ -86,15 +86,15 @@ and the data files in `/tmp/share`.
 
  ## Playing the game
  
- The game takes place on a rectangular areana where a snake continuously
+ The game takes place on a rectangular arena where a snake continuously
  move in one of the four directions: left, right, up and down --- it never 
- stops. As the snake moves it looses energy and if all of it is exausted, the
+ stops. As the snake moves, it looses energy, and if all this energy is exausted, the
  snake dies. To recover energy, the snake needs to eat pieces of food which
  are constantly replaced at random positions. 
 
 Be careful, though. The arena borders are electrified and would kill the snake
 if touched. Morover, mind that the snake is poisonous and it would also die if 
-it accidently bites itself, i.e. if the snake's head crosses its own body (yes, 
+it accidently bites itself, i.e. if the head of snake crosses its own body (yes, 
```

Comment:
I think it works better like this:

```suggestion
it accidently bites itself, i.e. if the it's head crosses its own body (yes, 
```

But if you'd rather keep the other way, there's a "the" missing:
```suggestion
it accidently bites itself, i.e. if the head of the snake crosses its own body (yes, 
```

Reply:
I liked the first option:

>it accidently bites itself, i.e. if it's head crosses it's own body (yes, 

But it's _its_ and not _it's_. There's no "is" there :P

Reply:
> But it's _its_ and not _it's_. There's no "is" there :P

TIL you shouldn't use the apostrophe for the possessive pronoun